modules/contracts-for-testing/src/main/kotlin/com/r3/corda/lib/tokens/testing/states/Rubles.kt

```diff
@@ -11,8 +11,8 @@ import net.corda.core.identity.AbstractParty
 import net.corda.core.transactions.LedgerTransaction
 
 @BelongsToContract(FungibleTokenContract::class)
-open class RubleToken(override val amount: Amount<IssuedTokenType<RUB>>,
-                      override val holder: AbstractParty) : FungibleToken<RUB>(amount, holder)
+open class RubleToken(amount: Amount<IssuedTokenType>,
```

Comment:
How come you are sub-classing `FungibleToken` here? Shouldn't we be sub-classing `TokenType` ?

Reply:
this is for a specific edge case in testing where I wanted to enforce that a `FungibleToken` cannot be moved to a `SubclassedFungibleToken` during a move